tests/pipeline/test_us_equity_pricing_loader.py

```diff
@@ -257,41 +253,46 @@
 )
 
 
-class MockDailyBarSpotReader(object):
-    """
-    A BcolzDailyBarReader which returns a constant value for spot price.
-    """
-    def spot_price(self, sid, day, column):
-        return 100.0
+class USEquityPricingLoaderTestCase(WithAdjustmentReader,
+                                    ZiplineTestCase):
+    SIM_PARAMS_START = TEST_CALENDAR_START
+    SIM_PARAMS_END = TEST_CALENDAR_STOP
+    asset_ids = 1, 2, 3
 
+    @classmethod
+    def make_equity_info(cls):
+        return EQUITY_INFO
 
-class USEquityPricingLoaderTestCase(TestCase):
+    @classmethod
+    def make_splits_data(cls):
+        return SPLITS
 
     @classmethod
-    def setUpClass(cls):
-        cls.test_data_dir = TempDirectory()
-        cls.db_path = cls.test_data_dir.getpath('adjustments.db')
-        all_days = TradingEnvironment().trading_days
-        cls.calendar_days = all_days[
-            all_days.slice_indexer(TEST_CALENDAR_START, TEST_CALENDAR_STOP)
-        ]
-        daily_bar_reader = MockDailyBarSpotReader()
-        writer = SQLiteAdjustmentWriter(cls.db_path, cls.calendar_days,
-                                        daily_bar_reader)
-        writer.write(SPLITS, MERGERS, DIVIDENDS)
+    def make_mergers_data(cls):
+        return MERGERS
 
-        cls.assets = TEST_QUERY_ASSETS
-        cls.asset_info = EQUITY_INFO
-        cls.bcolz_writer = SyntheticDailyBarWriter(
-            cls.asset_info,
-            cls.calendar_days,
+    @classmethod
+    def make_dividends_data(cls):
+        return DIVIDENDS
+
+    class make_adjustment_writer_daily_bar_reader(object):
```

Comment:
Everywhere else that we mock this out, we just return an instance of `MockDailyBarReader`.